### PR TITLE
[Service] handle dbus error case

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Check returned val and handle error case when failed to invoke dbus methods.

Signed-off-by: Jaeyun <jy1210.jung@samsung.com>

### CODE REVIEW

File: c/src/ml-api-service-agent-client.c [hunk 1]
```diff
@@ -262,12 +318,19 @@ ml_service_get_pipeline_state (ml_service_h h, ml_pipeline_state_e * state)
   }
 
   server = (_ml_service_server_s *) mls->priv;
-  machinelearning_service_pipeline_call_get_state_sync (mlsp, server->id,
-      &ret, &_state, NULL, NULL);
+  result = machinelearning_service_pipeline_call_get_state_sync (mlsp,
+      server->id, &ret, &_state, NULL, NULL);
```
Comment (gichan-jang) [inline c/src/ml-api-service-agent-client.c:322]:
```suggestion
      server->id, &ret, &_state, NULL, &err);
```

Comment (jaeyun-jung) [reply c/src/ml-api-service-agent-client.c]: updated

**Review by gichan-jang — APPROVED** [review]
LGTM

**Review by songgot — APPROVED** [review]
LGTM

**Pull request merged**